hooks/useRailgunProvider.tsx

```diff
@@ -1,19 +1,43 @@
 import { useEffect, useState } from 'react';
 import { setProviderForNetwork } from '@railgun-community/quickstart';
+import { BigNumber } from 'ethers';
 import { useNetwork, useProvider } from 'wagmi';
 import { networks } from '@/utils/networks';
 import { loadProviders } from '@/utils/railgun';
 
+// Fee is in bips, e.g. a value of 25 is a 0.25% fee.
+interface ShieldFee {
+  [chainId: number]: BigNumber;
+}
+
+const initialShieldFees: ShieldFee = {};
+Object.keys(networks).forEach((chainId) => {
+  // Current fees are 0.25% everywhere, so we initialize with that as a default -- If something goes
+  // wrong fetching fees, better to show a higher fee than a zero default fee.
```

Comment:
If something goes wrong fetching fees, i.e. the `loadProvider` call, the app can't work anyway though IIUC?

Reply:
in that case, the comment is a little misleading, because "if something goes wrong fetching fees," the app won't work (and won't show a higher fee than a zero default fee), right? might suggest truncating to:

```
// Current fees are 0.25% everywhere, so we initialize with that as a default 
```